src/System.IO/src/netstandard15aot/project.json

```diff
@@ -0,0 +1,19 @@
+{
+  "frameworks": {
+    "netstandard1.5": {
```

Comment:
Since this has been changed to `netstandard13aot`, is the commit message still valid?
The PR title and commit message: "Introducing `netstandard51aot` target.."